src/DoocsXy.cc

```diff
@@ -10,18 +10,18 @@ namespace ChimeraTK {
   : D_xy(doocsPropertyName, xValues->getNumberOfSamples(), eqFct), _xValues(xValues), _yValues(yValues) {
     if(_xValues->isReadable()) {
       updater.addVariable(
-          OneDRegisterAccessor<float>(xValues), eqFct, std::bind(&DoocsXy::updateValues, this, _xValues->getId()));
+          OneDRegisterAccessor<float>(xValues), eqFct, std::bind(&DoocsXy::updateDoocsBuffer, this, _xValues->getId()));
       _consistencyGroup.add(OneDRegisterAccessor<float>(xValues));
     }
 
     if(_yValues->isReadable()) {
       updater.addVariable(
-          OneDRegisterAccessor<float>(yValues), eqFct, std::bind(&DoocsXy::updateValues, this, _yValues->getId()));
+          OneDRegisterAccessor<float>(yValues), eqFct, std::bind(&DoocsXy::updateDoocsBuffer, this, _yValues->getId()));
       _consistencyGroup.add(OneDRegisterAccessor<float>(yValues));
     }
   }
 
-  void DoocsXy::updateValues(TransferElementID& elementId) {
+  void DoocsXy::updateDoocsBuffer(const TransferElementID& elementId) {
```

Comment:
I guess otherProperties are not updated because this variable is never used doocs->app?

Reply:
Yes.